### PR TITLE
docs:  第七章文档翻译

### PR DESCRIPTION
**Pull request by echoLC**

### CODE REVIEW

File: docs/pages/07-upload pictures/01-config server interface.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Config server interface
```
Comment (hahaaha) [inline docs/pages/07-upload pictures/01-config server interface.md]: 标题按之前讨论的首字母大写，下面也一样～

File: docs/pages/07-upload pictures/09-callback.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+```javascript
+editor.config.uploadImgHooks = {
+    // before upload
+    before: function(xhr) {
+        console.log(xhr)
+
+        // You can prevent users from uploading.
+        return {
+            prevent: true,
+            msg: '需要提示给用户的错误信息'
```
Comment (hahaaha) [inline docs/pages/07-upload pictures/09-callback.md]: 这里最好还是转换为英文

File: docs/pages/07-upload pictures/01-config server interface.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+editor.config.uploadImgServer = '/upload-img'
+
+editor.create()
+```
+
+The picture menu of the editor will display the tab and icon of the uploaded picture, after the configuration is complete, as shown in the figure below.
+
+![](../../images/upload-img.png)
+
+**It is important that the format of the server's return is!!!**
```
Comment (hahaaha) [inline docs/pages/07-upload pictures/01-config server interface.md]: 这个is是不是有点多余了。。仅供参考

File: docs/pages/07-upload pictures/11-save pictures in base64 format.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Save pictures in base64 format
+
+Set `editor.config.uploadImgShowBase64 = true` to save pictures in base64 format. That is, you can select a local picture, and the editor will display the picture in base64 format.
```
Comment (hahaaha) [inline docs/pages/07-upload pictures/11-save pictures in base64 format.md]: 这里set是不是要用ing形式

**Comment by shch** [comment]
01 check access => check passed 
that is return string => when return string
check not access -> check failed

**Comment by ilifewebdev** [comment]
you can see the developer tools of browser reading the error message => please open the browser developer tools and check the error message output by console.error

You can push types in uploadImgAccept array. => You can put the accepted types in uploadImgAccept array.

and I want to insert the image into the editor by yourself => and I want to insert the image into the editor by myself

上传到阿里云OSS 好像没有

**Comment by echoLC** [comment]
可能考虑到 老外应该不用阿里云吧，是否要翻译还要问下朋哥。

**Review by hahaaha — APPROVED** [review]

**Pull request merged**